### ISSUE TITLE
Phoenix 1.7.0-dev dialyzer warning for phx.gen.auth --live

### ISSUE THREAD

**Issue by cvkmohan** [phoenixframework/phoenix#4919]
When I am trying to create a new application from the master, when I run the generator 
```
mix phx.gen.auth Accounts User users --live
```
I am getting two dialyzer warnings - one in `user_registration_live.ex [Ln 18, Col 7]`
```
The test 
          <<_:144>> == 
          <<>> can never evaluate to 'true'
```
and another in `user_settings_live.ex [Ln 44, Col 7]`
```
The test 
          <<_:192>> == 
          <<>> can never evaluate to 'true'
```
Both of them related to the new `~p` sigil.

**Comment by optikfluffel** [comment]
This should be fixed in the current master branch, see #4916 for details.

**Comment by cvkmohan** [comment]
Thanks @optikfluffel. Though I created the app from latest master only, it somehow did not work for me. I will try again.  However, by changing the line to `action={~p"/users/log_in?#{%{_action: "registered"}}"}` removed the warning. That should suffice for now.
Thanks for the help.
I am closing this issue.
